src/api/app/models/event/base.rb

```diff
@@ -105,9 +107,10 @@ def receiver_roles
     end
 
     def initialize(attribs)
-      attributes = attribs.dup
+      attributes = attribs.dup.with_indifferent_access
       super()
       self.created_at = attribs[:time] if attributes[:time]
+      self.package = Package.get_by_project_and_name(attributes[:project], attributes[:package]) if attributes[:project] && attributes[:package]
```

Comment:
We should work this robuster, more bullet proof, in case project and/or package are remote. See https://github.com/openSUSE/open-build-service/wiki/Links

Reply:
Fixed